demo/server/src/modules/index.ts

```diff
@@ -21,28 +21,29 @@ const config = appConfig.getConfig();
 export let portalsecurityMiddleware: EndpointSecurity;
 export let mobileSecurityMiddleware: EndpointSecurity;
 
-// Setup all modules
+/**
+ * Setup modules for the mobile and portal app and mount it to
+ * the express app.
+ */
 export function setupModules(app: express.Express) {
   const mobileApp = express.Router();
-  const portalApp = express.Router();
-  portalsecurityMiddleware = securitySetup(portalApp, sessionOpts);
   mobileSecurityMiddleware = securitySetup(mobileApp);
   const connectionPromise = syncSetup(mobileApp);
+  demoDataSetup(connectionPromise);
+
+  const portalApp = express.Router();
+  portalsecurityMiddleware = securitySetup(portalApp, sessionOpts);
```

Comment:
As mentioned above we do not use interface. Let's just rename this to reflect what we really setting this up. This method is just helper to setup passport strategy (simplification, hiding complexity)